HMSegmentedControl/HMSegmentedControl.m

```diff
@@ -152,56 +152,57 @@ - (void)commonInit {
     self.borderWidth = 1.0f;
     self.badgeBackgroundColor = [UIColor redColor];
     self.badges = [NSArray array];
-    
```

Comment:
I think this will work since `_badgeYOffset` is a primitive value so it's "default" is zero, but it might be clearer to set it to zero explicitly.